source/02-qubits.Rmd

```diff
@@ -486,7 +486,7 @@ We could also step through the circuit diagram and follow the evolution of the s
 \begin{tikzpicture}
   \begin{scope}[xscale=3]
     \node (input) at (-1.5,0) {$\ket{0}$};
-    \node (inter) at (0,0) {$\frac{1}{2}\left[e^{i\frac{\pi}{4}}\ket{0}+e^{-i\frac{\pi}{4}}\ket{1}\right]$};
+    \node (inter) at (0,0) {$\frac{1}{\sqrt(2)}\left[e^{i\frac{\pi}{4}}\ket{0}+e^{-i\frac{\pi}{4}}\ket{1}\right]$};
```

Comment:
sorry but these need to be with `{}` instead of `()`! i.e. `\sqrt{2}`

Reply:
(and same for the other ones!)

Reply:
Of course! Should be done now I think :)